nixos/modules/services/web-apps/ocis.nix

```
@@ -0,0 +1,123 @@
+{
+  config,
+  lib,
+  pkgs,
+}:
+
+let
+  inherit (lib) types;
+  cfg = config.services.ocis;
+in
+
+{
+  options = {
+    services.ocis = {
+      enable = lib.mkEnableOption "Enable ownCloud Infinite Scale Stack.";
```

"Whether to enable" is added automatically, so this should only contain the name.

```suggestion
      enable = lib.mkEnableOption "ownCloud Infinite Scale";
```